basestar-schema/src/test/resources/io/basestar/schema/uniqueNamesInterface.yml

```diff
@@ -0,0 +1,23 @@
+Unique:
+  type: interface
+
+  properties:
+    my-name:
+      required: true
+      type: string
+    MyName:
+      type: integer
+    MySurname:
+      type: integer
+    first:
+      type: integer
+  links:
+    MY_SURNAME:
+      schema: A
+      expression: true
+
+A:
+  properties:
+    first:
```

Comment:
Do I understand correctly that `first` is conflicting in this schema? (from looking at assertions) If so, why is it conflicting - `Unique` and `A` are different schemas, and `links` is a soft-link instead of a nested explodable item?

Reply:
ah, good then - sorry for false alarm